src/main/java/com/sequenceiq/cloudbreak/service/cluster/flow/RecipeEngine.java

```diff
@@ -25,6 +25,7 @@
 @Component
 public class RecipeEngine {
 
+    public static final int DEFAULT_PLUGN_TIMEOUT = 15;
```

Comment:
please rename it to DEFAULT_RECIPE_TIMEOUT because timeout is related to recipes, not plugins